pkg/http/api.go

```diff
@@ -1323,42 +1329,54 @@ func (a *api) onDirectActorMessage(reqCtx *fasthttp.RequestCtx) {
 	policyRunner := resiliency.NewRunnerWithOptions(reqCtx, policyDef,
 		resiliency.RunnerOpts[*invokev1.InvokeMethodResponse]{
 			Disposer: resiliency.DisposerCloser[*invokev1.InvokeMethodResponse],
+			Accumulator: func(r *invokev1.InvokeMethodResponse) {
+				if r == nil {
+					return
+				}
```

Comment:
This is not necessary because the accumulator is invoked only if r is not zero (in this case, non-nil)